cmd/docker-app/bundle.go

```diff
@@ -22,7 +22,6 @@ import (
 
 type bundleOptions struct {
 	invocationImageName string
```

Comment:
I think we should remove this option. While building a local-only bundle, invocation-image name does not seem important to customize. And at push-time, it will anyway be pushed in the same repo as the bundle itself.

Reply:
(that would simplify the makeBundle function, and enforce coherence between bundle registry ref and invocation image registry ref)

Reply:
Done